pkgs/applications/radio/gnuradio/default.nix

```
@@ -60,7 +62,10 @@ let
         boost
         log4cpp
         mpir
-      ];
+      ]
+        # when gr-qtgui is disabled, icu needs to be included, otherwise
+        # building with boost 1.7x fails
+        ++ lib.optionals (!(hasFeature "gr-qtgui" features)) [ icu ];
```

```suggestion
        ++ lib.optionals !(hasFeature "gr-qtgui" features) [ icu ];
```

that should also work.

---

> that should also work.

Doesn't work.